kolibri/core/templates/kolibri/base.html

```diff
@@ -10,6 +10,7 @@
 {% block frontend_assets %}
 {% webpack_asset 'default_frontend' %}
 <script type="text/javascript" charset="utf-8">
+  var rootURL = "learn";
```

Comment:
For reference, I have worked out how to do this properly now. Will submit a PR to nav-feature when we have merged your PR here.